server/helpers/notification.js

```diff
@@ -0,0 +1,66 @@
+import models from '../models';
+import sendEmail from '../config/email';
+import template from './template';
+import { pusher } from '../config/pusher';
+
+const { User, Follower } = models;
+
+const notification = {
+  /**
+   *
+   * @param {*} articleTitle
+   * @param {*} authorId
+   * @returns {*} sends an email to the author
+   */
+  async sendEmailNotificationComment(articleTitle, authorId) {
+    const email = await User.findOne({
```

Comment:
You can do:

const authorEmail = await User.findOne({ ... })
The email variable is not necessary